int32 DIGIN_1 
float64 AIN0_1
float64 AIN1_1
